src/api/helpers/get-inactive-applications/index.test.ts

```diff
@@ -0,0 +1,91 @@
+import getInactiveApplications from '.';
+import { createFullApplication, getKeystoneContext } from '../../test-helpers';
+import { Application, Context } from '../../types';
+
+import { APPLICATION } from '../../constants';
+
+const { IN_PROGRESS, ABANDONED } = APPLICATION.STATUS;
+
+describe('api/helpers/get-inactive-applications', () => {
+  let context: Context;
+  let application4: Application;
+  let application5: Application;
+  let application6: Application;
+
+  const now = new Date();
+  const twoMonthsAgo = new Date(now.setDate(now.getDate() - 60));
+
+  const updateData = {
+    updatedAt: twoMonthsAgo,
+  };
+
+  beforeAll(async () => {
+    context = getKeystoneContext();
+  });
+
+  beforeEach(async () => {
+    await createFullApplication(context);
+    await createFullApplication(context);
+    await createFullApplication(context);
+    application4 = await createFullApplication(context);
+    application5 = await createFullApplication(context);
+    application6 = await createFullApplication(context);
+  });
```

Comment:
Can we create a test helper for this, i.e `createMultipleFullApplications`, so it's just a one liner

```js
const applications = await createMultipleFullApplications();
```